validate/validate.go

```diff
@@ -269,20 +269,20 @@ func TdxAttestation(quote *pb.QuoteV4, options *Options) error {
 
 // RawTdxQuoteValidate checks the raw bytes representation of an attestation quote.
 //
-// Deprecated: Use RawTdxAttestation instead. This function is no longer recommended for use.
+// Deprecated: Use RawTdxQuote instead. This function is no longer recommended for use.
```

Comment:
Please follow up with removing the deprecated functions.

Reply:
After renaming function names in go-tpm-tools, I'll remove deprecated functions.